in2code/femanager/CVE-2022-44543.yaml

```diff
@@ -0,0 +1,11 @@
+title: 'TYPO3-EXT-SA-2022-015: Broken Access Control in extension "femanager" (femanager)'
+link: 'https://typo3.org/security/advisory/typo3-ext-sa-2022-015'
+cve: CVE-2022-44543
+branches:
+    master:
```

Comment:
Should this be either `develop` or `main`?

Reply:
This should be `main`. I will update the PR